### DISCUSSION TITLE
Wechsel von OpenGTU mit Smart Micro Solar / Shelly EM3 auf OpenDTU on Battery

### DISCUSSION THREAD

**Discussion by MichaelHH71** [hoylabs/OpenDTU-OnBattery#545] [General]
Hallo,
erst einem vielen Dank für das Programmieren dieser Version der Open DTU!
Seit ein paar Tagen habe ich openDTU mit dem Smartt Micro Solar am laufen.
Ich würde gerne auf Basis der am WR anliegenden DC-Spannung den WR ein bzw. ausschalten.
Bisher zeigt mir das Dashboard der on Battery keine Spannung am WR an, das Status Feld ist orange. Habe auch schon einen Werksreset gemacht und alle Daten neue eingetragen,  leider ohne Erfolg.
Zudem die Frage, wie der Smart  Micro Solar eingebunden wird. Den Shelly EM3 habe ich an einer O2 box.

Schon mal vielen Dank!

Herzliche Grüße,
Michael

**Comment by MichaelHH71** [comment]
Inzwischen bin ich etwas weiter:
Den Shelly EM3+konnte ich dank der Wiki einbinden, so dass der Smart Micro Solar Controller
entfallen kann.
Im Power Limit habe ich Werte eingetragen, die für einen 8-Zellen LiFePo passen sollten.
20% Abschaltung = 24V
80% Einschalten = 27,2 V
Leistung min.. 10W
Leistung max. 150W
Sollwert 5 W
Hystereses 5 W
Passthrough 95 %

Problem: Der Wechselrichter schaltet sich beim Einschalten oder Neustart immer auf die 10W und dann nach 10-15sec wieder ab (Orange)
Das Protokoll zeigt an, dass der Regler durch externe Fernbedienung abgeschaltet wurde.

Gibt es dazu eine Erklärung?

**Reply by SW-Niko** [reply]
Wie hoch ist momentan deine LiFePo4 Spannung?
Ich vermute OpenDTU aktiviert den Wechselrichter erst wenn die Spannung über den eingestellten 27.2V liegt.
Schalte das Logging für den Powerlimiter ein und checke die Ausgabe in der Konsole.

**Reply by MichaelHH71** [reply]
Denke die Frage hat sich geklärt. Es ist scheinbar nicht möglich den WR manuell zu starten bevor
der Akku nicht die Zielspannung erreicht hat. In der Konsole konnte ich das im Logging gut nachvollziehen.
Insofern kann ich das System jetzt mal so laufen lassen.
Frage trotzdem: gibt es die Option das Entladen manuell zu starten, daher der DTU vorzugeben, dass sie gerade im Entladeteil ist sowie auch die Gegenrichtung, d.h. manuell vom Entladebetrieb wieder in den Ladebetrieb zu wechseln?
